setup.py

```diff
@@ -113,7 +114,7 @@ def alltests():
           'Topic :: Internet :: WWW/HTTP',
           'Framework :: Zope3',
           ],
-      url='http://pypi.python.org/pypi/zope.container',
+      url='http:///zopecontainer.readthedocs.io',
```

Comment:
We typically prefer the Github URL as the "project homepage," rather than the docs URL.